spec/gettext_i18n_rails/backend_spec.rb

```diff
@@ -77,16 +78,28 @@
       subject.translate('xx', 'c', {}).should == 'd'
     end
 
+    it 'temporarily sets the given locale' do
+      FastGettext.should_receive(:set_locale).with('xx').and_return('yz')
```

Comment:
I don't like the stubbing ether, but if I leave it out, the real caching is performed and all the other tests won't call :current_repository any longer. So this stubbing isolates the tests.